plugin/src/main/java/net/pl3x/map/configuration/Advanced.java

```diff
@@ -636,24 +641,24 @@ Setting a color to black (#000000) will make it invisible.""")
         put(Blocks.MANGROVE_WALL_SIGN, 0x763631);
         put(Blocks.MANGROVE_WOOD, 0x544329);
         put(Blocks.MEDIUM_AMETHYST_BUD, 0xA37CCC);
-        put(Blocks.MELON, 0x6D901E);
         put(Blocks.MELON_STEM, 0x9B9B9B);
```

Comment:
I think the stems can be removed since I color those on age. Would need to test it to be sure. I can do that later.